### ISSUE TITLE
[unpkg]: Handling of scoped npm packages

### ISSUE THREAD

**Issue by prafullbhosale** [aspnet/LibraryManager#156]
### Minimal repro steps
*What is the smallest, simplest set of steps to reproduce the issue. If needed, provide a project that demonstrates the issue.*  

1. Restore the below libman.json

```js
{
  "version": "1.0",
  "libraries": [
    {
      "provider": "unpkg",
      "library": "@angular/cli",
      "destination": "lib/angular"
    },
    {
      "provider": "unpkg",
      "library":"jquery",
      "destination": "lib/unpkg/jquery"
    }
  ]
}
```

### Expected result
*What would you expect to happen if there wasn't a bug*  
Both libraries succeed in restoring or both fail (as both are missing the version information)
In case of failure, the error should indicate that the library ids are not valid.

### Actual result
*What is actually happening*  
`@angular/cli` library is successfully restored and `jquery` fails to restore with a resource download exception. 

Error for jquery
```
Failed to download resource from "http://unpkg.com/jquery@/AUTHORS.txt"
```

**Comment by prafullbhosale** [comment]
This also affects auto-completion in the editor and `Install Client Side Library` dialog.

**Comment by alexgav** [comment]
I played around with this issue and it looks like various issues have been fixed already.
 
1. If I use the manifest file content from the repro, I do get errors about unpkg not being able to resolve the libraries (because versions are missing), which seems by design.

1. If I add version numbers to both libraries, they both resolve and restore correctly (and scoped package is now handled fine).

1. If I add version number to only one library (and omit version number from the other library), no libraries are restored even though only one of the two is invalid.
